app/stac_api/collection_spatial_extent.py

```diff
@@ -90,14 +87,22 @@ def update_bbox_extent(self, trigger, geometry, original_geometry, item):
                         },
                     )
                     start = time.time()
-                    qs = type(item).objects.filter(collection_id=self.pk).exclude(id=item.pk)
+                    raw_sql = '''
+                    SELECT
+                    1 AS id,
+                    st_AsText(st_extent(geometry)) AS geometry__extent
+                    FROM stac_api_item
+                    WHERE collection_id = %d
+                    AND NOT id = %d
+                    ''' % (self.pk, item.pk)
```

Comment:
Nice solution! You think you can manage to get rid of the raw sql by making use of https://docs.djangoproject.com/en/3.1/ref/models/expressions/#django.db.models.Func? Would be great if we could avoid raw_sql as much as possible